index.js

```diff
@@ -41,7 +42,9 @@ function rehype2react(options) {
       }
     }
 
-    return toH(h, node, settings.prefix);
+    var hast = tableCellStyle(node);
+
+    return toH(h, hast, settings.prefix);
```

Comment:
Any reason why this isn’t on one line?

Reply:
I've just followed the way of remark-react: https://github.com/mapbox/remark-react/blob/master/index.js#L88
It's completely no problem to do it with one line.

Reply:
It’s a minor not, it doesn’t matter much, but I’d prefer it one line I think, you?

Reply:
Yeah I'd respect your sense of beauty.